m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+  preprocessing_options.heading_list(1).conversion = @deg2rad;
+
+% $$$   preprocessing_options.time_list(2).time = 'Posixtime_nav';  
+% $$$   preprocessing_options.position_list(1).longitude = 'Lon';
+% $$$   preprocessing_options.position_list(1).latitude  = 'Lat';
+% $$$   preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  
+  %% CTD (each serie is a different configuration of glider sensors)
+  %  **All variables from a serie must be found to go forward**
+  preprocessing_options.ctd_list(1).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(1).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(1).pressure     = 'GPCTD_PRESSURE';
+  preprocessing_options.ctd_list(1).oxygen       = 'GPCTD_DOF';
+
+  preprocessing_options.ctd_list(2).conductivity = 'SBD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(2).temperature  = 'SBD_TEMPERATURE';
+  preprocessing_options.ctd_list(2).pressure     = 'SBD_PRESSURE';
+  preprocessing_options.ctd_list(2).oxygen       = 'SBD_DOF';
+
+  preprocessing_options.ctd_list(3).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(3).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(3).pressure     = 'GPCTD_PRESSURE';
+
+  %% Optics
+  preprocessing_options.optics_list(1).chlorophyll = 'FLBBCD_CHL_SCALED';
+  preprocessing_options.optics_list(1).cdom        = 'FLBBCD_CDOM_SCALED';
+  preprocessing_options.optics_list(1).backscatter_700 = 'FLBBCD_BB_700_SCALED';
+
+  preprocessing_options.optics_list(2).chlorophyll = 'TRI_CHL_SCALED';
+  preprocessing_options.optics_list(2).cdom        = 'TRI_CDOM_SCALED';
+  preprocessing_options.optics_list(2).backscatter_700 = 'TRI_BB_700_SCALED';
+
+  preprocessing_options.optics_list(3).chlorophyll = 'FLNTU_CHL_SCALED';
+  preprocessing_options.optics_list(3).turbidity = 'FLNTU_NTU_SCALED';  
+  
+  
+  %% MFL
+  preprocessing_options.mfl_list(1).mfl_v1 = 'MFL_V1';
+  preprocessing_options.mfl_list(1).mfl_v2 = 'MFL_V2';
+  preprocessing_options.mfl_list(1).mfl_v3 = 'MFL_V3';
+  preprocessing_options.mfl_list(1).mfl_v4 = 'MFL_V4';
+  preprocessing_options.mfl_list(1).mfl_te = 'MFL_TMPE';
+  preprocessing_options.mfl_list(1).mfl_td = 'MFL_TMPD';  
+
+  preprocessing_options.mfl_list(2).mfl_v1 = 'UV1_V1';
+  preprocessing_options.mfl_list(2).mfl_v2 = 'UV1_V2';
+  preprocessing_options.mfl_list(2).mfl_v3 = 'UV1_V3';
+  preprocessing_options.mfl_list(2).mfl_v4 = 'UV1_V4';  
+  preprocessing_options.mfl_list(2).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(2).mfl_td = 'UV1_TMP2';  
+
+  preprocessing_options.mfl_list(3).mfl_v1 = 'UV1_PHE';
+  preprocessing_options.mfl_list(3).mfl_v2 = 'UV1_TRY';
+  preprocessing_options.mfl_list(3).mfl_v3 = 'UV1_LD1';
+  preprocessing_options.mfl_list(3).mfl_v4 = 'UV1_LD2';
+  preprocessing_options.mfl_list(3).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(3).mfl_td = 'UV1_TMP2';  
+
```

Comment:
The first answer is the good one: a given set of raw measurements from the device
can be converted to different physical quantity (concentrations of one or another hydrocarbon). For example, Naphtalene and Tryptophan have about the same excitation/emission couple (270nm/340nm). The user must "deduce" (with other environmental information) whether the the signal is naphtalene-like (hydrocarbon) or Tryptophan-like (biological). Nobody never did science with this info (this my job for now!). So how we handle these variables is still an open question.